### PR TITLE
Documentation for SBOMs

### PR DESCRIPTION
**Pull request by dervoeti**
Fixes https://github.com/stackabletech/internal-issues/issues/46
It is explained how to browse through, manually validate and enforce the presence of SBOMs.

This won't yet work for release images (the current release does not have SBOMs). It also won't currently work for operator images coming from Nexus (docker.stackable.tech), this will be fixed by [upcoming PRs from operator-templating](https://github.com/stackabletech/airflow-operator/pull/389) (currently blocked though). For operator images in Harbor (like oci.stackable.tech/sdp/airflow-operator) it works.

This is the page:
https://deploy-preview-552--stackable-docs.netlify.app/home/nightly/tutorials/viewing-and-verifying-sboms

### CODE REVIEW

**Comment by dervoeti** [comment]
Thanks for the feedback! Fixed all of it. Also replaced underscores with hyphens in my other docs.

File: modules/tutorials/pages/viewing-and-verifying-sboms.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+[source,bash]
+----
+cosign verify-attestation --type cyclonedx
+--certificate-identity-regexp
+'^https://github.com/stackabletech/.+/.github/workflows/.+@.+'
+--certificate-oidc-issuer https://token.actions.githubusercontent.com
+docker.stackable.tech/stackable/airflow-operator:24.3.0
+| jq '.payload' -r | base64 -d | jq '.predicate'
+----
```
Comment (fhennig) [inline modules/tutorials/pages/viewing-and-verifying-sboms.adoc:25]:
```suggestion
----
cosign verify-attestation --type cyclonedx \
--certificate-identity-regexp \
'^https://github.com/stackabletech/.+/.github/workflows/.+@.+' \
--certificate-oidc-issuer https://token.actions.githubusercontent.com \
docker.stackable.tech/stackable/airflow-operator:24.3.0 \
| jq '.payload' -r | base64 -d | jq '.predicate'
----
```

The command wasn't copy-pastable anymore, this should fix it.

**Review by fhennig — CHANGES_REQUESTED** [review]
Just one thing, thanks for the changes!

**Review by fhennig — APPROVED** [review]
:+1:

**Pull request merged**